src/components/expandable-row.vue

```diff
@@ -73,25 +73,15 @@ export default {
   }
 
   .button-cell {
-    font-size: 20px;
```

Comment:
`ConnectionToForms` makes the button 16px, so I don't think we need this. `align-self` ends up working out as a way to vertically align.